components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -130,13 +130,14 @@ clean {
 ext.gleanGenerateMetricsAPI = {
     variant ->
 
-    def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
+    def sourceOutputDir =  "$buildDir/generated/source/glean/${variant.dirName}/kotlin"
```

Comment:
Looks good -- that old location always felt not-quite-right to me.

Reply:
For reference, I cargo culted the solution from [this google group](https://groups.google.com/forum/#!topic/adt-dev/jyp3LCaXTvs).